--- prelude-correlator-5.1.0/setup.py.orig	2020-02-26 18:17:53.001101760 +0100
+++ prelude-correlator-5.1.0/setup.py	2020-02-26 18:19:21.977286402 +0100
@@ -146,7 +146,7 @@
         ("etc/prelude-correlator/rules/python", [
             os.path.join('rules', x) for x in os.listdir('rules') if x.endswith('.py')
         ]),
-        ("var/lib/prelude-correlator/prelude-correlator", [
+        ("var/lib/prelude-correlator", [
             os.path.join('rules', x) for x in os.listdir('rules') if x.endswith('.dat')
         ])
     ],
